.vortex/installer/tests/Fixtures/install/names/scripts/custom/provision-10-example.sh

```
@@ -11,7 +11,7 @@
    # Note that deployment hooks for already enabled modules have run in the
    # parent "provision.sh" script.
    task "Installing custom site modules."
--  drush pm:install sw_core sw_search
+-  drush pm:install sw_base sw_search
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)